packages/trpc/server/routers/viewer/teams/inviteMember/inviteMember.handler.ts

```diff
@@ -138,43 +204,112 @@ export const inviteMemberHandler = async ({ ctx, input }: InviteMemberOptions) =
         })
       );
 
-      await sendTeamInviteEmails({
-        currentUserName: ctx?.user?.name,
+      await sendExistingUserTeamInviteEmails({
+        currentUserName: inviter.name,
         currentUserTeamName: team?.name,
         existingUsersWithMembersips: autoJoinUsers,
         language: translation,
         isOrg: input.isOrg,
         teamId: team.id,
+        isAutoJoin: true,
         currentUserParentTeamName: team?.parent?.name,
       });
     }
 
     // invited users cannot autojoin, create provisional memberships and send email
     if (regularUsers.length) {
-      await createProvisionalMemberships({
+      await createMemberships({
         input,
         invitees: regularUsers,
+        parentId: team.parentId,
+        accepted: false,
       });
-      await sendTeamInviteEmails({
-        currentUserName: ctx?.user?.name,
+      await sendExistingUserTeamInviteEmails({
+        currentUserName: inviter.name,
         currentUserTeamName: team?.name,
         existingUsersWithMembersips: regularUsers,
         language: translation,
         isOrg: input.isOrg,
         teamId: team.id,
+        isAutoJoin: false,
         currentUserParentTeamName: team?.parent?.name,
       });
     }
-  }
 
-  if (IS_TEAM_BILLING_ENABLED) {
-    if (team.parentId) {
-      await updateQuantitySubscriptionFromStripe(team.parentId);
-    } else {
-      await updateQuantitySubscriptionFromStripe(input.teamId);
```

Comment:
Are we still updating the subscription quantity?

Reply:
Yeah, no change in that. It's still there(https://github.com/calcom/cal.com/pull/13002/files/72cea4753ad73e0af717a488ae02fb800d6022fb#diff-95177a5fa5351ed5e907bdd95108677a14b23c59f5b214fd059d15750445b921R139). I just refactored out existing users handling.